apps/coordinator/tests/e2e.deploy.test.ts

```diff
@@ -264,10 +282,65 @@ describe("E2E Deployment Tests", () => {
         pollId: Number(pollId),
         approval,
         sessionKeyAddress,
-        chain: ESupportedNetworks.OPTIMISM_SEPOLIA,
+        chain: CHAIN,
       } as IMergeArgs),
     });
 
     expect(response.status).toBe(201);
   });
+
+  test.skip("should generate proofs correctly", async () => {
```

Comment:
I still have some `test.skip` here because I am running tests until we fix the `setVerifyingKeys` issue 😊